fractal/cmd/_workflow.py

```diff
@@ -81,20 +81,25 @@ async def post_workflowtask(
     *,
     project_id: int,
     workflow_id: int,
-    task_id_or_name: str,
+    task_id: Optional[int] = None,
+    task_name: Optional[str] = None,
```

Comment:
I think that also here we should have `task_version`, rather than `version` (see comment below).